content/agent/basic_agent_usage/heroku.md

```diff
@@ -41,66 +43,22 @@ In addition to the environment variables shown above, there are a number of othe
 | `DD_APM_ENABLED` | *Optional.* The Datadog Trace Agent (APM) is run by default. Set this to `false` to disable the Trace Agent. |
 | `DD_AGENT_VERSION` | *Optional.* By default, the buildpack installs the latest version of the Datadog Agent available in the package repository. Use this variable to install older versions of the Datadog Agent (note that not all versions of the Agent may be available). |
-| `DD_SERVICE_NAME` | *Optional.* While not read directly by the Datadog Agent, it is highly recommend that you set an environment variable for your service name. See the [Service Name](#service-name) section below for more information. |
-| `DD_SERVICE_ENV` | *Optional.* The Datadog Agent automatically tries to identify your environment by searching for a tag in the form `env:<environment name>`. For more information, see the [Datadog Tracing environments page][4]. |
+| `DD_SERVICE_ENV` | *Optional.* The Datadog Agent automatically tries to identify your environment by searching for a tag in the form `env:<environment name>`. For more information, see the [Datadog Tracing environments page][5]. |
 {{% /table %}}
 
-### Histogram percentiles
-
-Optionally set additional percentiles for your histogram metrics. By default only the 95th percentile is generated. To generate additional percentiles, set *all* percentiles, including the default, using the env variable `DD_HISTOGRAM_PERCENTILES`.  For example, to generate 0.95 and 0.99 percentiles, use the following command:
-
-```shell
-heroku config:add DD_HISTOGRAM_PERCENTILES="0.95, 0.99"
-```
-
-For more information about about additional percentiles, see the [percentiles documentation][5].
-
-### Service name
-
-A service is a named set of processes that do the same job, such as `webapp` or `database`. The service name provides context when evaluating your trace data.
-
-Although the service name is passed to Datadog on the application level, it is recommended that you set the value as an environment variable rather than directly in your application code.
-
-For example, set the service name as an environment variable:
-
-```shell
-heroku config:set DD_SERVICE_NAME=my-webapp
-```
-
-Then in a python web application, set the service name from the environment variable:
-
-```python
-import os
-from ddtrace import tracer
-
-service_nane = os.environ.get('DD_SERVICE_NAME')
-span = tracer.trace("web.request", service=service_name)
-...
-span.finish()
-```
-
-For Ruby on Rails applications, configure the `config/initializers/datadog-tracer.rb` file:
-
-```ruby
-require 'ddtrace'
-ies
-Datadog.configure do |c|
-  c.use :rails, service_name: ENV['DD_SERVICE_NAME'] || 'my-app'
-end
-```
-
-Setting the service name varies according to your language or supported framework. Reference the [Datadog libraries list][6] for specific language support.
+## More information
 
+For more information, view the source code, or contribute to this project, refer to the [Github project page][6].
```

Comment:
Could you please break this up into two sentences.

> For more information, view the source code or contribute to this project. Refer to the [Github project page][6].

Reply:
Hmm, I don't like my suggested phrasing, but could this sentence be broken up still, or rephrased?

Reply:
How about "Visit the Github project page for more information and to view the source code"?

I'm realizing that it's likely assumed that going to github is where contributing info would be, so maybe it doesnt need to be said.

Reply:
I like that a lot!